training_data_collector/main.py

```diff
@@ -0,0 +1,23 @@
+from operator import imod
+from selenium import webdriver
+from chessdotcom import actions
+from chessdotcom.labelGame import labelGame
+import time
+
+driver_location = "/snap/bin/chromium.chromedriver"
+binary_location = "/usr/bin/chromium-browser"
+
+options = webdriver.ChromeOptions()
+options.binary_location = binary_location
+options.add_argument("--user-data-dir=./user_data")
+
+driver = webdriver.Chrome(executable_path=driver_location, options=options)
+# driver.set_window_size(1280,720)
+game_id = "40420840437"
```

Comment:
Don't like this hardcoded but we can refactor it later on to take in a list of game ids or something, so fine for now.